docs/site/Routes.md

```diff
@@ -226,3 +226,40 @@ export class MyApplication extends RestApplication {
   }
 }
 ```
+
+## Mounting an Express Router
+
+If you have an
+[Express Router](https://expressjs.com/en/api.html#express.router) that you want
+to mount on top of a LoopBack application, you can use `mountExpressRouter`,
+which works on both `RestServer` and `RestApplication`.
+
+{% include note.html content="
+Make sure [express](https://expressjs.com/) is installed.
+" %}
+
+{% include code-caption.html content="src/index.ts" %}
+
+```ts
+import {RestApplication, RouterSpec} from '@loopback/rest';
+import {Request, Response} from 'express';
+import * as express from 'express';
+
+const app = new RestApplication();
+const router = express.Router();
+
+router.get('/pug', function(_req: Request, res: Response) {
+  res.send('Pug!');
+});
```

Comment:
Since our primary motivation for enabling this feature is to allow existing Express applications to be mounted in LB4, I would prefer if this example was more Express-app centric.

```ts
const legacyApp = express();
// your existing Express routes
legacyApp.get('/pug', function(_req: Request, res: Response) {
  res.send('Pug!');
});

// compose them with the new LB4 app
const spec: RouterSpec = {
  // insert your spec here, your 'paths', 'components', and 'tags' will be used
};
	
app.mountExpressRouter('/dogs', legacyApp, spec);
```

Also, I think `mountExpressRouter` should be typically called from the application constructor. To promote this best practice, I would prefer the example to use the same approach too.

```ts
const legacyApp = require('./express-app');

const legacySpec = {
   // insert your spec here, your 'paths', 'components', and 'tags' will be used
}

class MyApplication extends RestApplication {
  constructor(/*...*/) {
    // ...
   
    this.mountExpressRouter('/dogs', legacyApp, legacySpec);
  }
}
```